plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -242,11 +245,36 @@ func (s *SQLServer) Gather(acc telegraf.Accumulator) error {
 }
 
 func (s *SQLServer) gatherServer(server string, query Query, acc telegraf.Accumulator) error {
-	// deferred opening
-	conn, err := sql.Open("mssql", server)
-	if err != nil {
-		return err
+	// deferred connection opening
```

Comment:
nit: this comment should go below when opening the connection

Reply:
fixed